poms/bom/pom.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.datastax.oss.quarkus</groupId>
+    <artifactId>cassandra-quarkus-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+  <artifactId>cassandra-quarkus-bom</artifactId>
+  <packaging>pom</packaging>
+  <name>Cassandra Quarkus :: BOM :: Runtime</name>
+  <properties>
+    <cassandra-quarkus.version>1.0.0-SNAPSHOT</cassandra-quarkus.version>
+    <datastax-java-driver.version>4.5.1</datastax-java-driver.version>
+  </properties>
+  <dependencyManagement>
```

Comment:
1. Ok, I moved it to the runtime module `pom.xml`.
2. You are right - I've removed the definition of dependencies that are included in the `quarkus-parent` bom
3. I was also thinking that it may be a good idea because it will not cause a clash of dependencies if some module is not using the test dependency. Done.